hadoop-hdds/container-service/src/main/java/org/apache/hadoop/ozone/container/keyvalue/impl/BlockManagerImpl.java

```diff
@@ -162,8 +164,8 @@ public BlockData getBlock(Container container, BlockID blockID)
       }
       byte[] kData = db.getStore().get(Longs.toByteArray(blockID.getLocalID()));
       if (kData == null) {
-        throw new StorageContainerException("Unable to find the block." +
-            blockID, NO_SUCH_BLOCK);
+        throw new StorageContainerException(NO_SUCH_BLOCK_ERR_MSG + blockID,
+            NO_SUCH_BLOCK);
       }
```

Comment:
Would it make sense to go further and extract this to a method that looks up block by `blockID`?

Reply:
Done